BigCommerce/Integration/01_ManageBigCommerceConnection.md

```diff
@@ -0,0 +1,195 @@
+
+    ![Connection credentials](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/ConnectionCredentials.png "[Connection credentials]")
+
+2. Click the *Settings* tab.
+
+3. Click the *Variant option type* entry.   
+   The *Variant option type used for variants* dropdown is displayed.
+
+    ![Variant option type](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/VariantOptionType.png "[Variant option type]")
+
+4. Select the *Variant option type* you want to use.
+   The following option types are available:
+   | Dropdown | Meaning   
+   |-----|-------   
+   |Dropdown| An offer variant is selected in a dropdown
+   |Rectangles | An offer variant is selected in a small rectangle, that contains the offer variant value. For example:![Rectangle](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/VariantOptionRectangle2.png "[Rectangle]")
+    |Radio buttons | An offer variant is selected with a radio button.   
+
+5. Click the *Order filters* option.   
+   A list of order status is displayed.
+
+   ![Order filters](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/OrderFilters.png "[Order filters]")  
+
+6. Enable the toggles at the order statuses you want to import to the *Actindo Core1 Platform*. For detailed information on the *BigCommerce* order statuses, see the *BigCommerce* documentation under [https://developer.bigcommerce.com/docs/rest-management/orders/order-status#get-all-order-statuses](https://developer.bigcommerce.com/docs/rest-management/orders/order-status#get-all-order-statuses).
+
+7. If desired, enter the first order ID you want to import in the *Import only orders with an ID as of* field. This option is only relevant if you have orders in *BigCommerce* that should not be imported to *Actindo* in general.
+
+8. If desired, enter the date as of which you want to import the orders in the *Import only orders created as of* field. The date is valid from 00:00. This option is only relevant if you have orders in *BigCommerce* that should not be imported to *Actindo* in general.
+
+9. If desired, request the import of orders that have been already archived by enabling the *Import archived order* toggle.
+
+10. Click *Offers* option.
+
+   ![Offers option](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/OfferOption.png "[Offers option]")
+
+11. Enable the *Only push mapped attributes* toggle, if you want to import only those attributes that are mapped to the offer by the ETL data mapping. 
+
+12. Click the [Save] button.   
+  The connection settings are applied.
+
+
+
+## Manage ETL mappings 
+
+Create ETL mappings to map the *BigCommerce* attributes for subsequent processes in your *Actindo Core1 platform*. During connecting *BigCommerce* with *Actindo*, the *BigCommerce* attribute sets have been automatically created in the *DataHub* module.   
```

Comment:
```suggestion
Create ETL mappings to map the *BigCommerce* attributes for subsequent processes in your *Actindo Core1 platform*. While connecting *BigCommerce* with *Actindo*, the *BigCommerce* attribute sets have been automatically created in the *DataHub* module.   
```
Oder During the connection of BigCommerce with... 
https://dictionary.cambridge.org/de/grammatik/britisch-grammatik/during

Reply:
hat mir meine Übersetzung so beigebracht